.github/workflows/build-action.yml

```diff
@@ -1,145 +1,90 @@
-# yaml-language-server: $schema=https://json.schemastore.org/github-workflow
-
 name: Build
 
 on:
-  push:
-  pull_request:
+  workflow_call:
+    inputs:
```

Comment:
The following argument are a carbon copy of the `strategy` parameters previously used